extensions-core/lookups-cached-global/src/main/java/org/apache/druid/server/lookup/namespace/JdbcCacheGenerator.java

```diff
@@ -74,6 +86,14 @@
     final String keyColumn = namespace.getKeyColumn();
 
     LOG.debug("Updating %s", entryId);
+
+    boolean doIncrementalLoad = !Strings.isNullOrEmpty(namespace.getTsColumn())
+        && lastDBUpdate != null && lastVersion != null;
+
+    String sqlQuery = doIncrementalLoad ?
```

Comment:
Please use if-else instead of multiline ternary.

Reply:
sorry about that, fixed